connector/cargo_template/src/config.rs

```diff
@@ -0,0 +1,8 @@
+
+use fluvio_connector_common::connector;
+
+#[connector(config, name = "{{project-name}}")]
```

Comment:
the name here must be equal to a section name in the sample-config.yaml file which is `custom` now, so here should also be `custom` or not present at all (which will use `custom` by default)